### PR TITLE
Analytics Settings: remove EmptyContent and show nudge

### LINKED ISSUE #4567: Plans: update google analytics settings to show nudge and disabled fields

### PR DESCRIPTION
**Pull request by mtias**
image

Fixes #4567

### CODE REVIEW

**Comment by retrofox** [comment]
rebasing /editing some commits. Looks very weird.

image

**Comment by rralian** [comment]
This works well, code looks good. But I'd like to see the nudge show up for a premium plan site before we merge this. Right now it's missing the nudge and just shows the disabled form.

premium plan missing nudge

**Comment by retrofox** [comment]
> This works well, code looks good. But I'd like to see the nudge show up for a premium plan site before we merge this. Right now it's missing the nudge and just shows the disabled form.

done

File: client/my-sites/site-settings/form-analytics.jsx [hunk 1]
```diff
@@ -175,19 +196,25 @@ export default React.createClass( {
...
+
 		debug( 'Google analitics is not enabled. adding nudge ...' );
 
 		return (
 			<UpgradeNudge
 				title={ this.translate( 'Add Google Analytics' ) }
 				message={ this.translate( 'Upgrade to the business plan and include your own analytics tracking ID.' ) }
 				feature="google-analytics"
-				event="google_analytics_notice"
+				event={ eventName }
```
Comment (artpi) [inline client/my-sites/site-settings/form-analytics.jsx]: So I think we need something like this here:
```
href={ abtest( 'contextualGoogleAnalyticsNudge' ) ==='settingsDisabledFeature' ? '/plans/features/google-analytics/' + this.props.site.slug : '/plans/' + this.props.site.slug + '?feature=google-analytics' }
```

Comment (artpi) [inline client/my-sites/site-settings/form-analytics.jsx]: Maybe not ternary operator, but that would split the traffic...

Comment (artpi) [inline client/my-sites/site-settings/form-analytics.jsx]: `settingsDisabledFeature` should go to `/plans/features/google-analytics/artpitesting.wordpress.com`
`settingsDisabledPlansCompare` should go to `/plans/artpitesting.wordpress.com?feature=google-analytics`

**Comment by artpi** [comment]
Seems good, tested, but taking no responsibility whatsoever
:sheep: :ship:

**Pull request merged**